packages/components-react/src/paragraph/test/paragraph.spec.tsx

```diff
@@ -0,0 +1,33 @@
+import { describe, expect, it } from '@jest/globals';
+import { render, screen, within } from '@testing-library/react';
+import { LuxParagraph } from '../Paragraph';
+
+const paragraphText = 'Dit is een paragraaf';
+const leadParagraphText = paragraphText + ' die een inleiding is';
+const smallParagraphText = paragraphText + ' die klein is';
+
+describe('Paragraph', () => {
+  it('renders a paragraph', () => {
+    render(<LuxParagraph>{paragraphText}</LuxParagraph>);
+
+    expect(
+      within(screen.getByRole('paragraph')).getByText(paragraphText, { selector: '.utrecht-paragraph' }),
```

Comment:
Om de leesbaarheid te verhogen zou ik, waar mogelijk, die hele `within(...)` rits liever in een variabele zien. Zeker met tests is het belangrijk om duidelijk te zien wat er precies getest wordt.